apps/frontend/src/components/ui/input.tsx

```
@@ -3,17 +1,40 @@
+import * as React from 'react';
+import { cn } from '@/lib/utils';
+import { X } from 'lucide-react';
+import * as React from "react";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)